### PR TITLE
DOC-2411: Iframes in dialogs were not rendering rounded borders correctly.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2411

Site: [Staging branch](http://docs-feature-711-doc-2411tiny-10901.staging.tiny.cloud/docs/tinymce/latest/7.1.1-release-notes/#iframes-in-dialogs-were-not-rendering-rounded-borders-correctly)

Changes:
* added fix documentation for TINY-10901 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 1]
```diff
@@ -168,6 +168,15 @@ As a result, emoji's now render in all occasion without redirecting.
 
 For more information about these changes, see xref:emoticons.adoc#emoticons_images_url[Emoticons#emoticons_images_url].
 
+==== Iframes in dialogs were not rendering rounded borders correctly.
+// #TINY-10901
+
+Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected as the border radius was applied to the dialog and not the iframe.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:174]:
```suggestion
Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected, as the border radius was applied to the dialog and not the iframe.
```

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 2]
```diff
@@ -168,6 +168,15 @@ As a result, emoji's now render in all occasion without redirecting.
...
+
+{productname} {release-version} addresses this issue by applying the border radius to the `iframe` itself, ensuring that it renders correctly.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:176]:
```suggestion
{productname} {release-version} addresses this issue by also applying the border radius to the `iframe` itself, ensuring that it renders correctly.
```
The original statement implies that the border radius is no longer applied to the dialog.

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (spocke) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:174]: Maybe change `radius was applied to the dialog and not the iframe.` to `radius was applied to the iframe parent element and not the iframe.`

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:174]:
```suggestion
Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected as the border radius was applied to the `iframe` parent element and not the `iframe`.
```

**Pull request merged**